src/instructlab/utils.py

```diff
@@ -432,12 +432,15 @@ def validate_taxonomy(
         else:
             # Gather the new or changed YAMLs using git diff, including untracked files
             taxonomy_files = get_taxonomy_diff(taxonomy, taxonomy_base)
+        if not taxonomy_files:
```

Comment:
This is a change of behavior. This makes it an error condition for there to be no qna.yaml files. This seems a breaking change. Perhaps instead of an error, a polite message to the user that no qna.yaml files were found would be better.

Reply:
+1. @bjhargrave do you think a `TaxonomyReadingException` should be thrown here in that case?

Reply:
I don't think an exception should be raised. Just a message output should be sufficient.